### DISCUSSION TITLE
What do you want for the future of TODO to Issue?

### DISCUSSION THREAD

**Discussion by alstr** [alstr/todo-to-issue-action#209] [Ideas]
Having just released v5, I'm interested to find out what ideas people have for the future - whether that's the near future, or further down the line. The ideas can be grand, small, anything - I'm just interested to gauge the direction that people want things to move in. For instance, I was thinking about deeper customisation with advanced templates, or conditional parsing/issue creation. But I obviously want to work on what people want.

Thanks.

**Comment by terrancemacgregor** [comment]
Cool project.  I would love to add TODO's in .txt files as a default.   "Could not check "test-folder/test-file.txt" for TODOs as this language is not yet supported by default. "

**Reply by terrancemacgregor** [reply]
I wrote my own language.json and it works great.  Tried to add to the main project as a PR, could I couldn't get the tests work.  After fiddling with Docker image conflicts, python version,  and unit tests,  i decided not to push it because I couldn't commit enough time to make it super high quality.  Good luck!  

[
  {
    "language": "Plain Text",
    "extensions": [".txt"],
    "markers": [
      {
        "type": "line",
        "pattern": ""
      }
    ]
  }
]

**Reply by alstr** [reply]
Thanks! If you do want to resubmit a PR you can set `SKIP_PROCESS_DIFF_TEST` to `True` to skip some of the less relevant tests that can be problematic.
